locales/en/bento.ftl

```diff
@@ -0,0 +1,10 @@
+bento-button-title = Firefox apps and services
```

Comment:
Can we use the brand names from app.ftl here? e.g. `{ -brand-name } apps and services`.